core/src/test/java/org/apache/druid/math/expr/ParserTest.java

```diff
@@ -172,6 +173,43 @@ public void testMixed()
     validateFlatten("min(1, max(3, 4))", "(min [1, (max [3, 4])])", "1");
   }
 
+  @Test
+  public void testBitwiseOps()
+  {
+    validateFlatten("3 & 1", "(& 3 1)", "1");
+    validateFlatten("3 & 1", "(& 3 1)", "1");
```

Comment:
nit: duplicate test, did you intend to test the inverse of the previous line here?
```suggestion
    validateFlatten("1 & 3", "(& 1 3)", "1");
```

Reply:
Ah, it was just an accidental dupe I think